tests/main.js

```diff
@@ -2253,18 +2253,24 @@ test('Creates a manifest file with the list of created functions if the `manifes
   })
 })
 
-testMany('Correctly follows node_modules via symlink', ['bundler_esbuild', 'todo:bundler_nft'], async (options, t) => {
-  const fixtureName = 'node-module-symlinks'
-  const opts = merge(options, {
-    basePath: join(FIXTURES_DIR, fixtureName),
-  })
-  const { tmpDir } = await zipNode(t, fixtureName, {
-    opts,
-  })
+const skipOnWindows = (name) => (platform === 'win32' ? `skip:${name}` : name)
 
-  const isEven = require(`${tmpDir}/function`)
-  t.is(isEven(2), '2 is even')
-})
+testMany(
+  'Correctly follows node_modules via symlink',
+  ['bundler_esbuild', skipOnWindows('bundler_nft')],
```

Comment:
[dust] We have a few places in the test where we change the behaviour based on `platform`, and we're using simple `if` statements. I would rather keep that simplicity than adding an abstraction on top of it, but if we do insist on adding an abstraction, we should probably change the existing tests to use it.

Reply:
our existing usage of if statements skips entire `testMany` blocks, while here I want to only disable one variation. `if` statements would become a bit unwieldly here. I'll see what I can do to clarify things.

Reply:
Tests handlers receive the `variation` parameter. You could skip the test `if (variation === 'bundler_nft' && platform === 'win32')`?

Reply:
know what? scrap it. this is an escape hatch, we neither expect nor encourage to use this often. might as well make hard to type. 1f9b1054a85c7cd49e98dbf064a67f494d8c590a